onnxruntime/core/providers/cuda/tensor/pad_impl.cu

```diff
@@ -230,6 +230,6 @@ void PadNCHWInputWithPaddingAlongHAndWImpl(
 SPECIALIZED_IMPL(float)
 SPECIALIZED_IMPL(double)
 SPECIALIZED_IMPL(half)
-
+SPECIALIZED_IMPL(bool)
```

Comment:
Do we have test for bool type Pad? 
pad_test.cc?

Reply:
We have a test added for CPU bool type.
https://github.com/microsoft/onnxruntime/pull/7856/files